core/config.js

```diff
@@ -0,0 +1,25 @@
+Nuxeo = Nuxeo || {};
+Nuxeo.UI = Nuxeo.UI || {};
+Nuxeo.UI.config = Nuxeo.UI.config || {};
+
+Object.assign(Nuxeo.UI.config, {
+  get(path) {
+    return path.split('.').reduce((a, b) => a && a[b], this);
+  },
+  set(path, value) {
+    const parentPath = path.substring(0, path.lastIndexOf('.'));
+    let parent = this.get(parentPath);
+    if (!parent) {
+      parent = path
+        .split('.')
+        .slice(0, -1)
+        .reduce((a, b) => {
+          a[b] = a[b] || {};
+          return a[b];
+        }, this);
+    }
+    parent[path.substring(path.lastIndexOf('.') + 1)] = value;
+  },
+});
+
+export default Nuxeo.UI.config;
```

Comment:
Would it make sense to export `Nuxeo.UI` altogether and make this the default module so we could `import { config } from '@nuxeo/nuxeo-elements'` ?

Reply:
It seems weird to me export the whole Nuxeo.UI object from config.js, but we could export config as a named export instead of default. Do you prefer `import { config } from '@nuxeo/nuxeo-elements'` instead of `import config from '@nuxeo/nuxeo-elements'`?

Reply:
I guess it's more future proof as it'll allow us to export more stuff in the future

Reply:
but we can export both defaults and named and do something like `import config, { otherStuff1, otherStuff2 } from '../config.js';`, but perhaps it is uglier?

Reply:
idea was to have it in the default module file `index.js` so we could `import { config } from '@nuxeo/nuxeo-elements'` instead of `import { config } from '@nuxeo/nuxeo-elements/config.js'` which seems a bit redundant. This way we could start hidding more details of the file structure of the `nuxeo-elements` package.